css/styles.css

```diff
@@ -874,3 +891,120 @@ li.skills-title {
     color: #000;
   }
 }
+
+/* popup windows styles */
+#popup-window {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  top: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.95);
+  backdrop-filter: blur(0.5px);
+  z-index: 3;
+  justify-content: center;
+  align-items: center;
+}
+
+.popup-card {
+  height: 92vh;
+  width: 90vw;
+  padding: 3vw 3vh;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  gap: 0.8em;
+}
+
+.popup-card-header {
+  display: flex;
+  flex-direction: column;
+  gap: 0.7em;
+}
+
+.popup-card-body {
+  height: 60%;
+  margin-top: 1.5em;
+}
+
+.popup-card-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.popup-card h3 {
+  font-family: 'Roboto', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 1.5em;
+  color: #172b4d;
+}
+
+.popup-card-tags {
+  list-style: none;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 0.8em;
+  line-height: 1.3em;
+  color: #3c3a39;
+  display: flex;
+  gap: 0.8em;
+}
+
+.popup-card-tags .tag {
+  margin: 0;
+}
+
+.popup-card .popup-card-img {
+  height: 80%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
[OPTIONAL] Great job making the changes requested, I think it would be better if we use a `<img> `tag for our project Image and not use background image as it is a part of information we need to display to the user and not part of styling for which we use background image, It will definitely improve our site's accessibility too. :+1: